CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [5.0.1] - 2017-05-09
```

Comment:
This is what I am looking for we add the date of when it was released not when it was committed since we know that from git history and releases are manually kicked off by maintainers (yes I have raised the issue to automate this).
```
+### Changed
 +- check-elb-nodes.rb returns critical instead of unknown if total number of nodes equals zero
```